apps/admin-portal/src/app.tsx

```diff
@@ -105,6 +109,34 @@ export const App = (): ReactElement => {
             });
     }, [ config ]);
 
+    /**
+     * Set the application settings of the user to the local storage.
+     */
+    useEffect(() => {
+        if (userName === "") {
```

Comment:
Wouldn't it be better to check all the falsy values .

```suggestion
        if (!userName) {
```